Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1667,13 +1685,20 @@ open class NavigationMapView: UIView {
         return candidates
     }
     
-    private func routes(closeTo point: CGPoint) -> [Route]? {
-        let tapCoordinate = mapView.mapboxMap.coordinate(for: point)
-        
+    /**
+     Returns a list of the routes, that are close to a certain point and are within threshold distance
+     defined in `NavigationMapView.tapGestureDistanceThreshold`.
```

Comment:
`tapGestureDistanceThreshold` is typed as a `CGFloat`, but maybe its documentation needs to clarify that it’s measured in screen points, not meters.

Reply:
Makes sense, updated.